specification/network/resource-manager/Microsoft.Network/stable/2018-06-01/applicationGateway.json

```diff
@@ -0,0 +1,2142 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "NetworkManagementClient",
+    "description": "The Microsoft Azure Network management API provides a RESTful set of web services that interact with Microsoft Azure Networks service to manage your network resources. The API has entities that capture the relationship between an end user and the Microsoft Azure Networks service.",
+    "version": "2018-06-01"
```

Comment:
If changes are being ported from 06 to 04, there shouldn't be a `2018-06-01` directory in this PR